linux_os/guide/system/auditing/auditd_configure_rules/audit_privileged_commands/audit_rules_privileged_commands_pam_timestamp_check/rule.yml

```diff
@@ -10,11 +10,13 @@ description: |-
     configured to use the <tt>augenrules</tt> program to read audit rules during
     daemon startup (the default), add a line of the following form to a file with
     suffix <tt>.rules</tt> in the directory <tt>/etc/audit/rules.d</tt>:
-    <pre>-a always,exit -F path=/usr/sbin/pam_timestamp_check -F auid&gt;={{{ auid }}} -F auid!=unset -F key=privileged</pre>
+    <pre>-a always,exit -F path={{% if product == "sle12" %}}/sbin/pam_timestamp_check{{% else %}}/usr/sbin/pam_timestamp_check{{% endif %}}
```

Comment:
It would be better to refactor it so that the file path is extracted to a variable. That would prevent the multiple ifs and elses and it would mean more readable code as a result.

Reply:
Thanks, get addressed and update the PR.

Reply:
Done